runtime/src/genesis_utils.rs

```diff
@@ -200,6 +200,18 @@ pub fn activate_all_features(genesis_config: &mut GenesisConfig) {
     }
 }
 
+pub fn activate_all_features_except(
+    genesis_config: &mut GenesisConfig,
+    features: &HashSet<Pubkey>,
```

Comment:
nit: A more descriptive name than `features` might be good, and potentially eliminate the need for a comment. Something like `features_to_skip` maybe ?